python/lsst/dbb/buffmngrs/endpoint/ingester/ingester.py

```diff
@@ -170,7 +170,7 @@ def run(self):
                 if not self.daemon:
                     logger.debug(msg + "terminating.")
```

Comment:
(discovered by running pylint)  Lines 171 and 173 shouldn't use string concatenation, but lazy formatting to create the debug messages.